packages/autorest.go/src/m4togocodemodel/clients.ts

```diff
@@ -261,7 +261,11 @@ function adaptResponseEnvelope(m4CodeModel: m4.CodeModel, codeModel: go.CodeMode
     if (!modelType) {
       throw new Error(`didn't find type name ${resultProp.schema.language.go!.name} for response envelope ${respEnv.name}`);
     }
-    respEnv.result = new go.ModelResult(modelType, adaptResultFormat(helpers.getSchemaResponse(op)!.protocol));
+    const resultFormat = adaptResultFormat(helpers.getSchemaResponse(op)!.protocol);
```

Comment:
Do you want to call it serdeFormat here to be consistent throughout the code. We refer to it as SerDeFormat elsewhere in the code

Reply:
The resultant type is `go.ResultFormat` which is a superset of `SerDeFormat` (includes `Text`).